### PR TITLE
Uncapitalize type if trying to find translation-string

### PR DESCRIPTION
**Pull request by ghost**
related to #6378

### CODE REVIEW

**Comment by jhass** [comment]
Ah, good catch. Let's just downcase the whole thing though.

**Comment by Zauberstuhl** [comment]
w00t not sure why I made that so complicated ... maybe it was too early

**Comment by jhass** [comment]
Cool, squash?

File: app/mailers/report_mailer.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def self.new_report(type, id)
   def new_report(type, id, role)
     resource = {
       url:  report_index_url,
-      type: I18n.t("notifier.report_email.type." + type),
+      type: I18n.t("notifier.report_email.type." + type.downcase),
```
Comment (jhass) [inline app/mailers/report_mailer.rb]: Wait, actually update this to `"notifier.report_email.type.#{type.downcase}"`, then squash.

**Comment by Zauberstuhl** [comment]
done

**Pull request merged**